controllers/news.controller.js

```diff
@@ -42,4 +42,12 @@ exports.getTopics = (req, res, next) => {
     .catch((err) => {
         next(err);
     })
-  };
\ No newline at end of file
+  };
+
+  exports.commentCounter = (req, res, next) => {
```

Comment:
I don't think this is needed, as all of our comment counting work is now being done in our sql statement in the model.

Reply:
Whoops, remnants of some testing I forgot to remove